src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -32,7 +33,7 @@ void BulkReader::run() {
     m_stop = 0;
     unsigned char data[255];
 
-    while (m_stop.load() == 0) {
+    while (atomicLoadAcquire(m_stop) == 0) {
```

Comment:
@daschuer I think aquire the right strategy here?

Reply:
Yes. Thanks.